src/utils.ts

```diff
@@ -727,3 +731,16 @@ export const composeEventHandlers = <E>(
     }
   };
 };
+
+export const isOnlyExportingSingleFrame = (
+  elements: readonly NonDeletedExcalidrawElement[],
+) => {
+  const frames = elements.filter((element) => element.type === "frame");
+
+  return (
+    frames.length === 1 &&
+    elements.every(
+      (element) => element.type === "frame" || element.frameId === frames[0].id,
+    )
+  );
```

Comment:
is this still applicable or legacy check? AFAIK if a frame is selected, no other elements are. So we should only need to check if `elements.length === 1 && elements[0].type === "frame"`.

Reply:
When exporting, elements in the frame are supplied as well. So if we're indeed exporting a single frame, then every element is either the frame itself or the elements in the frame. If we have some elements outside the frame to export as well, the check will be false.